src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Add support for Filter expression
```

Comment:
Also add link to this PR. [e.g.](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md#:~:text=Update%20the%20ActivitySource.Name%20from%20%22OpenTelemetry.SqlClient%22%20to%20%22OpenTelemetry.Instrumentation.SqlClient%22.%20(%233435))

Reply:
updated